multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -103,6 +107,17 @@ def parse_genome_results(self, f):
                     d[k] = r_search.group(1)
             else:
                 d[k] = r_search.group(1)
+
+    for k, r in regexes_secondary.items():
+        r_search = re.findall(r, f["f"], re.MULTILINE)
+        print(r_search)
+        if len(r_search) > 1:
```

Comment:
Why > 1 and not just true?

Reply:
I think it was to make sure to only parse this if there is actually a second hit in the file (otherwise you would end up picking up the first line and including that again...) but I again, I'll leave this to Thiseas to confirm...

Reply:
exactly what @jfy133 said. The output contains two values named the same in two separate sections, with the on-target section (`Globals inside`) only appearing when a bedfile is given. So the first element of the search is already picked up by the original regex, but not the second.

Reply:
oof 🤦🏻‍♂️  http://tallphil.co.uk/writing-good-log-files/ :shakefist:

Reply:
`oof` was precisely my reaction when I saw that :laughing: . Is this otherwise a reasonable solution (and can resolve the comment?)

Reply:
Please leave it for now, I might have a quick play and see if I can refactor a bit. To be honest, if it's just finding the first hit and there can be repeats of things, it makes me a bit nervous that the original code could accidentally report the wrong thing in certain cases. So I'm inclined to refactor it to know which section it's in.

Reply:
Ok, refactored to parse per-section instead. Hopefully the code is a bit more robust to repeat strings now 👍🏻